anti-analysis/anti-vm/vm-detection/check-for-sandbox-username.yml

```diff
@@ -0,0 +1,60 @@
+rule:
+  meta:
+    name: check for sandbox username
+    namespace: anti-analysis/anti-vm/vm-detection
+    author: "@_re_fox"
+    scope: function
+    examples:
+      - ccbf7cba35bab56563c0fbe4237fdc41:0x402B90
+  features:
+    - and:
+      - api: GetUserName
+      - or:
+        - string: /MALTEST/i
+          description: Betabot Username Check  
+        - string: /TEQUILABOOMBOOM/i
+          description: VirusTotal Sandbox
+        - string: /SANDBOX/i
+          description: Gookit Username Check 
+        - string: /^VIRUS/i
+          description: Satan Username Check
+        - string: /MALWARE/i
+          description: Betabot Username Check
+        - string: /SAND\sBOX/i
```

Comment:
Thanks for the git notes.  I'll use them going forward.  

Most of the work that I contribute up is copy and pasted from inside a docker container, so there is a bit of a lack of "real" working environment.  I tend just to use the website, it doesn't have a great workflow, but it's easy enough.  That does lead to some odd behaviour though.  Often deleting branches and re-forking of repos due to my container no longer being around.  Most of this is due to working in VM's  where I don't have python2 or vivisect installed.

In terms of using `\s`.  I picked up my instructors bad habits.  If it's easier to read one way or the other, I'm happy to adjust.

Reply:
@re-fox 

> Most of the work that I contribute up is copy and pasted from inside a docker container, so there is a bit of a lack of "real" working environment. I tend just to use the website, it doesn't have a great workflow, but it's easy enough. That does lead to some odd behaviour though. Often deleting branches and re-forking of repos due to my container no longer being around. Most of this is due to working in VM's where I don't have python2 or vivisect installed.

I tried to setup capa in my Linux computer a few days ago and I had problems using both Python2 and Python3 with [pyenv](https://github.com/pyenv/pyenv) (I didn't want to install the dependencies as system libraries with root). I was thinking that a docker container may be a good idea, because our current development setup (which uses both python 2 and 3) breaks other commonly used workflows (such as pyenv). I'll give this a thought and see if I come up with a good solution for this. 😉 

> In terms of using `\s`.  I picked up my instructors bad habits.  If it's easier to read one way or the other, I'm happy to adjust.

I find an space easier to read, but it is ok. I was just curious if there was another reason for it. Thanks for the explanation! 😄